lib/active_record/connection_adapters/cockroachdb/schema_statements.rb

```diff
@@ -296,31 +295,6 @@ def type_to_sql(type, limit: nil, precision: nil, scale: nil, array: nil, **) #
           sql
         end
 
-        # This overrides the method from PostegreSQL adapter
-        # Resets the sequence of a table's primary key to the maximum value.
-        def reset_pk_sequence!(table, pk = nil, sequence = nil)
```

Comment:
the difference lied in a condition `database_version < 10`, which will never be true in our codebase!